crates/bevy_a11y/src/lib.rs

```diff
@@ -99,9 +99,9 @@ pub enum AccessibilitySystem {
 }
 
 /// Plugin managing non-GUI aspects of integrating with accessibility APIs.
-pub struct AccessibilityPlugin;
+pub struct A11YAccessibilityPlugin;
```

Comment:
Imo this should stay the same. I’d expect `bevy_a11y` to export THE accessibility plugin.

Reply:
I agree, since a11y means accessibility, that would mean "accessibility accessibility plugin".

Reply:
Okay, it makes sense.